src/statics/icons/user.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="36.099" height="41.256" viewBox="0 0 36.099 41.256"><defs><style>.a{opacity:0.25;}.b{fill:#ef7d18;}</style></defs><g class="a"><path class="b" d="M146.5,3.021a10.314,10.314,0,1,1-14.586,0,10.314,10.314,0,0,1,14.586,0" transform="translate(-121.156 0)"/><path class="b" d="M66.093,295.891a28.648,28.648,0,0,0-31.642,0,5.114,5.114,0,0,0-2.228,4.258v8.937h36.1v-8.937A5.114,5.114,0,0,0,66.093,295.891Z" transform="translate(-32.223 -267.829)"/></g></svg>
```

Comment:
Just wondering if we need this or if we could use the one of Quasar?
It has Fontawesome and Material Icons
https://quasar.dev/vue-components/icon#Svg-icons

Reply:
this is just a placeholder for the real image of users, i can use one of fontawesome if u wont

Reply:
users already have a default placeholder image as base64 encoded data in the database AFAIK